.github/workflows/smoke-tests.yml

```diff
@@ -45,8 +45,8 @@ jobs:
     - name: Python Checkstyle
       timeout-minutes: 20
       run: |
-        pylint --disable=F0401 --disable=W0511 --disable=E1101 --disable=R0801 --disable=R0903  ./src/*/*.py
-        # disabled warnings: import loading, todos, function members, duplicate code, public methods
+        pylint --disable=F0401 --disable=W0511 --disable=E1101 --disable=R0801 --disable=R0903 --disable=R0915 ./src/*/*.py
```

Comment:
pylint was throwing this warning since userservice.create_app > 50 lines. We can refactor userservice into a [blueprint](https://flask.palletsprojects.com/en/1.1.x/blueprints/) style approach but I feel that may make it unnecessarily complicated.

Reply:
Yes, more details from official docs here about [factories](https://flask.palletsprojects.com/en/1.1.x/patterns/appfactories/#factories-extensions) and usage with [gunicorn](https://flask.palletsprojects.com/en/1.1.x/deploying/wsgi-standalone/#gunicorn)